backend/src/apiserver/config/sample_config.json

```diff
@@ -7,7 +7,7 @@
   {
     "name":"[Sample] ML - TFX - Taxi Tip Prediction Model Trainer",
     "description":"Example pipeline that does classification with model analysis based on a public tax cab BigQuery dataset. For source code, refer to https://github.com/kubeflow/pipelines/tree/master/samples/tfx",
```

Comment:
How about add both gcp and on-repm tar.gz to the UI?

Reply:
+1

Reply:
> How about add both gcp and on-repm tar.gz to the UI?

Done